src/formatting/expr.rs

```diff
@@ -496,7 +505,12 @@ fn block_prefix(context: &RewriteContext<'_>, block: &ast::Block, shape: Shape)
                     )?
                 )
             } else {
-                "unsafe ".to_owned()
+                match context.config.control_brace_style() {
```

Comment:
This should be `brace_style` not `control_brace_style`
https://rust-lang.github.io/rustfmt/?version=master&search=#brace_style
https://rust-lang.github.io/rustfmt/?version=master&search=#control_brace_style
```suggestion
                match context.config.brace_style() {
```

Reply:
I was unsure of the right setting (as unsafe is not listed in either category), will change this